eZ/Publish/API/Repository/Values/URL/URLQuery.php

```diff
@@ -47,4 +47,11 @@ class URLQuery extends ValueObject
      * @var int
      */
     public $limit = 25;
+
+    /**
+     * If true, search engine should perform count even if that means extra lookup.
+     *
+     * @var bool
+     */
+    public $performCount = true;
```

Comment:
I guess `\eZ\Publish\API\Repository\Values\URL\SearchResult::$totalCount` docblock should be updated as well to `int|null`

Reply:
Changed in caf54feb.

I didn't change the default value. Technically it's our internal VO and there is little chance that anyone creates those on their own, but for the sake of BC I've left it as is.

In case of our code we always pass the proper value in `totalCount` field in constructor, even if it's `null`.